### PR TITLE
Update tr.m3u

### PR DESCRIPTION
**Pull request by recepuncu**
Cease access to infringing live TV channels of Warner Bros. Discovery content on iptv-org.github.io.

### CODE REVIEW

**Review by BellezaEmporium — APPROVED** [review]

File: streams/tr.m3u [hunk 1]
```diff
@@ -76,8 +76,6 @@ http://stream.taksimbilisim.com:1935/ciftcitv/smil:ciftcitv.smil/playlist.m3u8
 https://live.artidijitalmedya.com/artidijital_ciftcitv/ciftcitv/playlist.m3u8
 #EXTINF:-1 tvg-id="Cine5.tr",Cine5 (720p)
 https://cdn-cine5tv.yayin.com.tr/cine5tv/cine5tv/playlist.m3u8
-#EXTINF:-1 tvg-id="CNNTurk.tr",CNN Türk (720p)
-https://live.duhnet.tv/S2/HLS_LIVE/cnnturknp/playlist.m3u8
```
Comment (Alstruit) [inline streams/tr.m3u:80]: Works.

**Review by Alstruit — CHANGES_REQUESTED** [review]

**Comment by recepuncu** [comment]
Immediately cease access to infringing live TV channels featuring Warner Bros. Discovery content on iptv-org.github.io. Failure to comply with this notice will result in legal action.

File: streams/tr.m3u (hunk 1 above)
Comment (recepuncu) [reply streams/tr.m3u:80]: Yes, it works. But the problem is that it is a violation of rights.

**Comment by Alstruit** [comment]
> Immediately cease access to infringing live TV channels featuring Warner Bros. Discovery content on iptv-org.github.io. Failure to comply with this notice will result in legal action.

https://github.com/iptv-org/iptv?tab=readme-ov-file#legal

File: streams/tr.m3u (hunk 1 above)
Comment (recepuncu) [reply streams/tr.m3u:80]: The following address has been closed to public access. I kindly ask you to check again.

https://live.duhnet.tv/S2/HLS_LIVE/cnnturknp/playlist.m3u8

**Comment by recepuncu** [comment]
The following address has been closed to public access. I kindly ask you to check again.

https://live.duhnet.tv/S2/HLS_LIVE/cnnturknp/playlist.m3u8

**Pull request closed without merging**